src/domain/ledger/fee-reimbursement.ts

```diff
@@ -0,0 +1,13 @@
+import { toSats } from "@domain/bitcoin"
+
+export const FeeReimbursement = (prepaidFee: Satoshis): FeeReimbursement => {
+  const getReimbursement = ({ actualFee }: { actualFee: Satoshis }): Satoshis | null => {
```

Comment:
why to deal with two types when you can just return 0? it does not break any rule and is easier and safe to validate

Reply:
Fine with me

Reply:
I was thinking to return `null` when the checks break (if `feeDifference < 0` or `> max_fee`). Returning `0` when the actual difference isn't 0 might be misleading?

Reply:
Ah right - this reasoning also makes sense to me. I think either is fine. From the domain perspective `null` is standing in for some kind of `noReimbursementRequiredMarker`. But `0` is also fine to stand in for that because it literaly means `no reimbursement`